pkg/operator/controller.go

```diff
@@ -66,6 +66,10 @@ func NewController(kubeconfig *restclient.Config) (*Controller, error) {
 
 	p := parameters.Globals{}
 
+	p.Service.Name = imageregistryv1.ImageRegistryName
+	p.Service.Port = 443
```

Comment:
Having this as a parameter may give wrong feeling that this a parameter that can be changed. But at the same time we have `fmt.Sprintf("%s.%s.svc", svc.Name, svc.Namespace)` that makes this value impossible to change without additional modifications. It worth mention that we rely on the fact that this port is the default port Docker.

Reply:
I can drop this (in which case I can drop the TargetPort as well, since that should be the port on the container)